masterlist.yaml

```
@@ -20395,6 +20395,26 @@ plugins:
       - crc: 0x3DC7A6FB
         util: 'TES4Edit v4.0.4c'
 
+  - name: 'FrostcragAlchemySorter.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/13782/'
+        name: 'Frostcrag Alchemy Sorter'
+      - link: 'https://www.nexusmods.com/oblivion/mods/49969/'
+        name: 'Frostcrag Alchemy Sorter Updated'
+    msg:
+      - <<: *obsolete
+        type: error
```

```suggestion
      - <<: *patchUpdateAvailable
        type: warn
```

`patchUpdateAvailable` would be better here as I don't believe this is an official update. An error message is also a bit excessive for issue that the mod fixes.

---

I'm not sure if it would be, since Frostcrag Alchemy Sorter Updated isn't a patch but a replacement for Frostcrag Alchemy Sorter. The message for `obsolete` seems more appropriate, although even it isn't really accurate. Perhaps a custom message should be used here.

---

I just went through a bunch of uses of `patchUpdateAvailable` and it seems to be overloaded with meanings -- in some cases being used for replacement mods like this, and in some places used to point out fix patches for the original mod to be applied on top.

I'll switch over to `patchUpdateAvailable` but that overloading of meaning should probably get cleaned up at some point too.

---

If there's really a case where the message causes confusion due to its ambiguity, we could reconsider it, but I'd like to avoid changing or adding messages unless absolutely necessary. Doing so means throwing out & requiring new translations, which is not a simple task as we don't have a team of translators for every language we support on standby.